web/examples/craft/main.dart

```diff
@@ -93,3 +99,22 @@ void startCraft() {
     print("$fps fps, "+world.debugString());
   });
 }
+
+/// Returns the seed provided by the URL's query parameters, or null if no seed is found.
+int _getSeed() {
+  String seedQueryParam = Uri.base.queryParameters["seed"];
+  if (seedQueryParam == null) {
+    return null;
```

Comment:
Since `math.Random().nextInt` can only natural numbers (0 or more) and since this method is has a return type of `int`, I would recommend returning -1 instead of null to keep the return types as strict as possible (trying to not let the playdoughness of JS seep into Dart too much 😄) 
```suggestion
    return -1;
```